too long string in IBPSA.Utilities.Cryptographics.Validation.SHA1

For the example `IBPSA.Utilities.Cryptographics.Validation.SHA1`, JModelica correctly prints the warning
```
sources/IBPSA_Utilities_Cryptographics_Validation_SHA1_init_independent.c:56:5: warning: string length '510' is greater than the length '509' ISO C90 compilers are required to support [-Woverlength-strings]
     JMI_ASG(STR_Z, _s_pi_strIn5_4, ("aaaaaa...
```
with a few thousand characters 'a'. 

The Modelica Language definitions states "Support for external functions written in C (specifically C89)" and C89 has a limit of 509 characters in a character string literal or wide string literal (after concatenation).

I therefore suggest to change the example to use a smaller string.

--- a/IBPSA/Utilities/Cryptographics/Validation/SHA1.mo
+++ b/IBPSA/Utilities/Cryptographics/Validation/SHA1.mo
@@ -11,8 +11,7 @@ model SHA1 "Model that verifies the SHA1 encryption C function"
     "Third test string";
   parameter String strIn4 = "1.23e+4"
     "Fourth test string";
-  parameter String strIn5 = Modelica.Utilities.Strings.repeat(1000,
-    string="aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
+  parameter String strIn5 = Modelica.Utilities.Strings.repeat(509, string="a")
     "Fifth test string";
 
   //Expected outputs
@@ -29,7 +28,7 @@ model SHA1 "Model that verifies the SHA1 encryption C function"
     "bdd220adb45b392f17915af70ed8a006c382b983"
     "Encryption result of fourth string";
   parameter String strEx5=
-    "34aa973cd4c4daa4f61eeb2bdbad27316534016f"
+    "edff7a135c2e06d4c8084e61b4516c901bd5fcd0"
     "Encryption result of fifth string";
 
   //Comparison results
@@ -65,7 +64,7 @@ the following strings:
 <code>&#34;1.23e+4&#34;</code>
 </li>
 <li>
-<code>&#34;a&#34;</code> repeated a million consecutive times
+<code>&#34;a&#34;</code> repeated <i>509</i> times.
 </li>
 </ul>
 <p>
@@ -75,6 +74,11 @@ results, the <code>cmpAll</code> boolean variable will be <code>true</code>.
 </html>",
 revisions="<html>
 <ul>
+<li>
+January 13, 2018, by Michael Wetter:<br/>
+Reduced string length for
+<a href=\"https://github.com/ibpsa/modelica-ibpsa/issues/1081\">issue 1081</a>.
+</li>
 <li>
 May 30, 2018, by Alex Laferri&egrave;re:<br/>
 First implementation.

--- a/IBPSA/Utilities/Cryptographics/Validation/SHA1.mo
+++ b/IBPSA/Utilities/Cryptographics/Validation/SHA1.mo
@@ -75,7 +75,7 @@ results, the <code>cmpAll</code> boolean variable will be <code>true</code>.
 revisions="<html>
 <ul>
 <li>
-January 13, 2018, by Michael Wetter:<br/>
+January 13, 2019, by Michael Wetter:<br/>
 Reduced string length for
 <a href=\"https://github.com/ibpsa/modelica-ibpsa/issues/1081\">issue 1081</a>.
 </li>

